core/src/main/java/io/undertow/server/HttpServerExchange.java

```diff
@@ -1851,27 +1851,15 @@ public StreamSinkConduit wrap(ConduitFactory<StreamSinkConduit> factory, HttpSer
      * @return <code>true</code> if reads or writes were resumed
      */
     boolean runResumeReadWrite() {
-        boolean ret = false;
-        if(anyAreSet(state, FLAG_SHOULD_RESUME_WRITES)) {
+        if(isResumed()) {
             responseChannel.runResume();
-            ret = true;
-        }
-        if(anyAreSet(state, FLAG_SHOULD_RESUME_READS)) {
-            requestChannel.runResume();
```

Comment:
This does not look correct, the request channel is not resumed anymore.

Reply:
Ah, sorry, I read this incorrectly. updated